.travis.yml

```diff
@@ -5,7 +5,7 @@ os: linux
 dist: xenial
 env:
   global:
-    - CONTAINER=centos:7
+    - CONTAINER=registry.centos.org/centos:7
```

Comment:
this is not needed

Reply:
I do not understand. In the travis.yml file, you can see the docker pull command. In case I do not change this variable, the docker pull command fails on the limit pull limit rate.

Reply:
@vinzenz Or do you mean to drop the docker pull command as well? As I understand, the explicit pull is not necessary and then we could drop the CONTAINER envar completely.

Reply:
Nevermind, I forgot that it is a bit different in leapp-repository